src/legacy/ui/public/directives/field_name/field_type_name.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { i18n } from '@kbn/i18n';
+
+export function fieldTypeName(type: string) {
+  switch (type) {
+    case 'boolean':
+      return i18n.translate('common.ui.directives.fieldNameIcons.booleanAriaLabel', {
+        defaultMessage: 'Boolean field',
+      });
+    case 'conflict':
+      return i18n.translate('common.ui.directives.fieldNameIcons.conflictFieldAriaLabel', {
+        defaultMessage: 'Conflicting field',
+      });
+    case 'date':
+      return i18n.translate('common.ui.directives.fieldNameIcons.dateFieldAriaLabel', {
+        defaultMessage: 'Date field',
+      });
+    case 'geo_point':
+      return i18n.translate('common.ui.directives.fieldNameIcons.geoPointFieldAriaLabel', {
+        defaultMessage: 'Geo Point',
```

Comment:
You're right, this doesn't follow our own guidelines 👍 
could you rename it to **Geo point field**, thx!